tests/test_utils.py

```diff
@@ -306,3 +313,513 @@ def check_multistream_classic(fnames):
+            skiprows=0,
+            delim_whitespace=True,
+            header=None,
+            names=names,
+            na_values=-9999.0000)
+
+    ecflux_df['DATES'] = pd.to_datetime(ecflux_df.YEAR * 10000 +
+                                 ecflux_df.MONTH * 100 +
+                                 ecflux_df.DAY,format='%Y%m%d')
+
+    ecflux_df.index = ecflux_df.DATES
+
+    ecflux_df.drop(['YEAR', 'DAY'], inplace=True, axis=1)
+
+    if ('HOUR' not in ecflux_df) and ('SEC' in ecflux_df):
+        # add hour column for groupby
+        ecflux_df['HOUR'] = (ecflux_df['SEC'] * (1/3600)
+                                        ).astype(int)
+
+    return(ecflux_df)
+
+def plot_fluxnet_comparison(driver, science_test_data_dir,
+                            compare_data_dict,
+                            result_dir, plot_dir,
+                            plots_to_make):
+
+    ''' makes Ameriflux figures
+    '''
+
+    # loop over Ameriflux sites
+    obs_dir = os.path.join(science_test_data_dir,
+                            'datasets',
+                            'ec_flux_towers',
+                            'obs')
+
+    for subdir in os.listdir(obs_dir):
+
+        if check_site_files(obs_dir, subdir):
+            # get CSV file from site directory to get lat/lng for site
+            lat, lng = get_fluxnet_lat_lon(obs_dir, subdir)
+
+            # loop over data to compare
+            data = {}
+            for key, items in compare_data_dict.items():
+
+                if key == "ecflux":
+                    try:
+                        # load Ameriflux data
+                        data[key] = read_fluxnet_obs(subdir,
+                                                science_test_data_dir, items)
+                    except OSError:
+                        warnings.warn("this site does not have data")
+
+                elif key == "VIC.4.2.d":
+                    try:
+                        # load VIC 4.2 simulations
+                        data[key] = read_vic_42_output(lat, lng,
+                                                science_test_data_dir,
+                                                items)
+
+                    except OSError:
+                        warnings.warn("this site has a lat/lng precision issue")
+
+                else:
+                    try:
+                        # load VIC 5 simulations
+                        data[key] = read_vic_5_output(lat, lng,
+                                                science_test_data_dir,
+                                                items)
+                    except OSError:
+                        warnings.warn("this site has a lat/lng precision issue")
+
+            # make figures
+            vic_vars = ['OUT_LATENT', 'OUT_SENSIBLE', 'OUT_SWNET', 'OUT_LWNET']
+            variable_names = ['LH', 'H',
+                                'SW NET',
+                                'LW NET']
```

Comment:
I might suggest putting a dictionary at the top of this module that is a mapping from VIC variable names to the labels we want to use on plots.  Then it can be reused by other comparisons.

Also good to do for units.